docpad.coffee

```diff
@@ -48,7 +48,6 @@ module.exports =
       'speakers'
       'schedule'
       'callforpapers'
-      'sponsors'
```

Comment:
This is supposed to be here, to suggest that anyone can be a sponsor (with the _your logo_ example image and link)

Reply:
Ok, I'll keep showing it.